server/boot/model-explorer.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function mountLoopBackExplorer(server) {
+  var explorer;
+  try {
+    explorer = require('loopback-explorer');
+    var restApiRoot = server.get('restApiRoot');
+
+    var explorerApp = explorer(server, {basePath: restApiRoot});
+    server.use('/explorer', explorerApp);
```

Comment:
hm... this isn't going to be the right location, is it?

I wonder if `smm = require('strong-mesh-models')` should expose:

- app() (no middleware, route starts at `/`, so pm can mount where it wants... maybe `/v1`, maybe not)
- server() (middleware, explorer, and start/stop methods)
- explorer() (this...)

so that the explorer can be mounted where it makes sense to the encapsulating app...